sortinghat/core/db.py

```diff
@@ -676,6 +698,34 @@ def move_identity(trxl, identity, uidentity):
     return uidentity
 
 
+def set_lock_unique_identity(trxl, uidentity, lock):
```

Comment:
I think we should define two functions `lock` and `unlock`. These two functions will call this one with the right parameters. Thus, it's easier to understand and functions in higher levels don't need to know what value should given to `lock` in case they want to lock or unlock the identity.

In addition, it would be clearer to rename `lock` value to  `lock_flag` or similar. `lock` sounds you are either locking or passing a lock.

Reply:
Done.